### PR TITLE
DM-44621: Fix current event could fetch old event

### PR DESCRIPTION
**Pull request by ugyballoons**
Also fixes view for no current event.

### CODE REVIEW

File: python/lsst/ts/rubintv/models/models.py [hunk 1]
```diff
@@ -150,6 +150,11 @@ class Event:
     filename: str = ""
     ext: str = ""
 
+    def __eq__(self, other: Any) -> bool:
+        if type(other) is not type(self):
+            raise TypeError
+        return self.key == other.key
+
```
Comment (sebastian-aranda) [inline python/lsst/ts/rubintv/models/models.py:157]: If I understand this correctly, this is being used by the `max` function to retrieve the event with the biggest key? I've tried to search some references but only find some stackoverflow comments explaining that the `max`function only needs `__lt__` or `__gt__` depending on the case. Is the `__eq__` really needed? Also, can you add some docstrings to both `__eq__` and `__lt__` explaining why they are needed, and why only those two?

**Review by sebastian-aranda — APPROVED** [review]
Looks good to me :+1:. I've just left one comment, which might be mostly a doubt than a requested change, hence I'm just approving the PR and leaving it to you.

File: python/lsst/ts/rubintv/models/models.py (hunk 1 above)
Comment (ugyballoons) [reply python/lsst/ts/rubintv/models/models.py:157]: You're right. The `__eq__` method was left over from a previous implementation. 
I'll remove it and add a docstring to the other dunder.

Comment (sebastian-aranda) [reply python/lsst/ts/rubintv/models/models.py]: thanks :pray:.

**Pull request merged**